mac/qwerty-fr.bundle/Contents/Resources/qwerty-fr.keylayout

```diff
@@ -191,18 +191,18 @@
             <key code="32" action="11"/>
             <key code="33" output="{"/>
             <key code="34" action="8"/>
-            <key code="35" output="P"/>
+            <key code="35" action="P"/>
             <key code="36" output="&#x000D;"/>
-            <key code="37" output="L"/>
-            <key code="38" output="J"/>
+            <key code="37" action="L"/>
+            <key code="38" action="J"/>
             <key code="39" output="&#x0022;"/>
-            <key code="40" output="K"/>
+            <key code="40" action="K"/>
             <key code="41" output=":"/>
             <key code="42" output="|"/>
             <key code="43" output="&#x003C;"/>
             <key code="44" output="?"/>
             <key code="45" action="9"/>
-            <key code="46" output="M"/>
+            <key code="46" action="M"/>
```

Comment:
Not quite sure why Ukelele replaced `output` with `action` here.

Reply:
Never mind. That's normal: the output when state is none is moved in the corresponding block.